specification/maps/data-plane/Microsoft.Maps/Weather/preview/1.0/weather.json

```diff
@@ -702,6 +702,305 @@
           }
         }
       }
+    },
+    "/weather/climo/actuals/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Climo Actuals**\n  \n  \n**Applies to**: S0 and S1 pricing tiers.\n\n\nThe service returns climatology data such as past daily actuals for a given coordinate location.",
+        "operationId": "Weather_GetDailyClimoActuals",
+        "x-ms-examples": {
+          "DailyClimoActuals": {
+            "$ref": "./examples/GetDailyClimoActuals.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionKey"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/JsonFormat"
+          },
+          {
+            "name": "query",
+            "description": "Coordinates of the location for which current conditions information is requested. The applicable query is specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
+            "in": "query",
+            "type": "string",
+            "required": true
+          },
+          {
+            "name": "startDate",
+            "description": "Start date in ISO 8601 format, for example, 2019-10-27.",
+            "in": "query",
+            "type": "string",
+            "format": "date",
+            "required": true
+          },
+          {
+            "name": "endDate",
+            "description": "End date in ISO 8601 format, for example, 2019-10-28.",
+            "in": "query",
+            "type": "string",
+            "format": "date",
+            "required": true
+          },
+          {
+            "$ref": "#/parameters/WeatherUnit"
+          },
+          {
+            "$ref": "#/parameters/WeatherLanguage"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DailyClimoActualsResponse"
+            }
+          },
+          "400": {
```

Comment:
Status codes listed in the `responses` section indicate a successful response, so the generated SDKs won't treat these as errors.  If there's a need to individually model these error responses you need to annotate _all_ error responses with [`x-ms-error-response`](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-error-response).